app/views/courses/index.html.erb

```diff
@@ -27,5 +27,14 @@
     <li>copy content between courses</li>
   </ul>
 
-  <%= render partial: 'course_table', locals: { courses: @provider.courses } %>
+  <%= render partial: 'course_table', locals: { courses: @self_accredited_courses.sort_by(&:name) } %>
```

Comment:
Can `.sort_by(&:name)` be put in the controller, please?

Reply:
Done